flink-core/src/main/java/org/apache/flink/api/common/state/StateTtlConfig.java

```diff
@@ -204,16 +215,32 @@ public Builder neverReturnExpired() {
 		 * Sets the time characteristic.
 		 *
 		 * @param timeCharacteristic The time characteristic configures time scale to use for ttl.
+		 *
+		 * @deprecated will be removed in a future version in favor of {@link #setTtlTimeCharacteristic}
 		 */
+		@Deprecated
 		@Nonnull
 		public Builder setTimeCharacteristic(@Nonnull TimeCharacteristic timeCharacteristic) {
-			this.timeCharacteristic = timeCharacteristic;
+			/**
```

Comment:
This looks like a doc comment, i would suggest to use `//` in inline comments or start it with `/*` instead of `/**`.